### ISSUE TITLE
On MacOSX, closing window still leaves app open, which then leads to error messages when user tries to say load a csv file

### ISSUE THREAD

**Issue by mattRedBox** [ODIQueensland/data-curator#652]
See: 
#611 

```
This should be a quick fix - was meant to remove the code that distinguished between:

on mac: closing window, but still leaving app open
on windows: closing everything
The behaviour should be to close everything (so that the behaviour is consistent across OSs).
The error you're seeing is because the logic that follows assume that there is always a window open.
Will add an issue and fix for release 0.14.0
```

**Comment by Stephen-Gates** [comment]
Fixed on macOS in [v0.14.0](https://github.com/ODIQueensland/data-curator/releases/tag/v0.14.0) 🎉

**Comment by louisjasek** [comment]
Nice work, all fixed - closing the window closed the entire application
